diff -burNp audacious-plugins-2.1/src/mtp_up/mtp.c audacious-plugins-2.1-dud/src/mtp_up/mtp.c
--- audacious-plugins-2.1/src/mtp_up/mtp.c	2009-07-07 00:40:36.000000000 +0200
+++ audacious-plugins-2.1-dud/src/mtp_up/mtp.c	2009-08-23 13:29:56.601303553 +0200
@@ -197,6 +197,7 @@ gint upload_file(Tuple *from_tuple)
     g_print("Uploading track '%s'\n",filename);
 #endif
     gentrack->parent_id = mtp_device->default_music_folder;
+    gentrack->parent_id = parent_id;
     ret = LIBMTP_Send_Track_From_File(mtp_device, filename , gentrack, NULL , NULL);
     LIBMTP_destroy_track_t(gentrack);
     if (ret == 0) 
