--- audacious-plugins-2.5.0/src/ladspa/ladspa.c~	2011-04-16 18:10:10.000000000 +0200
+++ audacious-plugins-2.5.0/src/ladspa/ladspa.c	2011-05-04 13:17:32.846667174 +0200
@@ -222,7 +222,7 @@
     if (ladspa_path == NULL)
     {
 	/* Fallback, look in obvious places */
-	find_plugins("/usr/lib/ladspa");
+	find_plugins("LIBDIR/ladspa");
 	find_plugins("/usr/local/lib/ladspa");
     }
     else
--- audacious-plugins-2.5.0/src/ladspa/Makefile~	2011-04-16 18:10:10.000000000 +0200
+++ audacious-plugins-2.5.0/src/ladspa/Makefile	2011-05-04 13:18:35.223333840 +0200
@@ -8,5 +8,5 @@
 plugindir := ${plugindir}/${EFFECT_PLUGIN_DIR}
 
 CFLAGS += ${PLUGIN_CFLAGS}
-CPPFLAGS += ${PLUGIN_CPPFLAGS} ${MOWGLI_CFLAGS} ${GTK_CFLAGS} ${GLIB_CFLAGS} ${GMODULE_CFLAGS} -I../..
+CPPFLAGS += ${PLUGIN_CPPFLAGS} ${MOWGLI_CFLAGS} ${GTK_CFLAGS} ${GLIB_CFLAGS} ${GMODULE_CFLAGS} -I../.. -DLIBDIR=${libdir}
 LIBS += ${GTK_LIBS} ${GLIB_LIBS} ${GMODULE_LIBS}
