--- a/m4/buildsys.m4~	2018-08-09 00:41:00.000000000 +0200
+++ b/m4/buildsys.m4	2018-08-09 17:58:09.579552969 +0200
@@ -28,73 +28,14 @@ AC_DEFUN([BUILDSYS_INIT], [
 		AC_SUBST(OBJC_DEPENDS, $GOBJC)
 		AC_SUBST(OBJCXX_DEPENDS, $GOBJCXX)
 
-		AC_PATH_PROG(TPUT, tput)
-
-		AS_IF([test x"$TPUT" != x""], [
-			if x=$($TPUT el 2>/dev/null); then
-				AC_SUBST(TERM_EL, "$x")
-			else
-				AC_SUBST(TERM_EL, "$($TPUT ce 2>/dev/null)")
-			fi
-
-			if x=$($TPUT sgr0 2>/dev/null); then
-				AC_SUBST(TERM_SGR0, "$x")
-			else
-				AC_SUBST(TERM_SGR0, "$($TPUT me 2>/dev/null)")
-			fi
-
-			if x=$($TPUT bold 2>/dev/null); then
-				AC_SUBST(TERM_BOLD, "$x")
-			else
-				AC_SUBST(TERM_BOLD, "$($TPUT md 2>/dev/null)")
-			fi
-
-			if x=$($TPUT setaf 1 2>/dev/null); then
-				AC_SUBST(TERM_SETAF1, "$x")
-				AC_SUBST(TERM_SETAF2,
-					"$($TPUT setaf 2 2>/dev/null)")
-				AC_SUBST(TERM_SETAF3,
-					"$($TPUT setaf 3 2>/dev/null)")
-				AC_SUBST(TERM_SETAF4,
-					"$($TPUT setaf 4 2>/dev/null)")
-				AC_SUBST(TERM_SETAF6,
-					"$($TPUT setaf 6 2>/dev/null)")
-			dnl OpenBSD seems to want 3 parameters for terminals
-			dnl ending in -256color, but the additional two
-			dnl parameters don't seem to do anything, so we set
-			dnl them to 0.
-			elif x=$($TPUT setaf 1 0 0 2>/dev/null); then
-				AC_SUBST(TERM_SETAF1, "$x")
-				AC_SUBST(TERM_SETAF2,
-					"$($TPUT setaf 2 0 0 2>/dev/null)")
-				AC_SUBST(TERM_SETAF3,
-					"$($TPUT setaf 3 0 0 2>/dev/null)")
-				AC_SUBST(TERM_SETAF4,
-					"$($TPUT setaf 4 0 0 2>/dev/null)")
-				AC_SUBST(TERM_SETAF6,
-					"$($TPUT setaf 6 0 0 2>/dev/null)")
-			else
-				AC_SUBST(TERM_SETAF1,
-					"$($TPUT AF 1 2>/dev/null)")
-				AC_SUBST(TERM_SETAF2,
-					"$($TPUT AF 2 2>/dev/null)")
-				AC_SUBST(TERM_SETAF3,
-					"$($TPUT AF 3 2>/dev/null)")
-				AC_SUBST(TERM_SETAF4,
-					"$($TPUT AF 4 2>/dev/null)")
-				AC_SUBST(TERM_SETAF6,
-					"$($TPUT AF 6 2>/dev/null)")
-			fi
-		], [
-			AC_SUBST(TERM_EL, '\033\133K')
-			AC_SUBST(TERM_SGR0, '\033\133m')
-			AC_SUBST(TERM_BOLD, '\033\1331m')
-			AC_SUBST(TERM_SETAF1, '\033\13331m')
-			AC_SUBST(TERM_SETAF2, '\033\13332m')
-			AC_SUBST(TERM_SETAF3, '\033\13333m')
-			AC_SUBST(TERM_SETAF4, '\033\13334m')
-			AC_SUBST(TERM_SETAF6, '\033\13336m')
-		])
+			AC_SUBST(TERM_EL, '')
+			AC_SUBST(TERM_SGR0, '')
+			AC_SUBST(TERM_BOLD, '')
+			AC_SUBST(TERM_SETAF1, '')
+			AC_SUBST(TERM_SETAF2, '')
+			AC_SUBST(TERM_SETAF3, '')
+			AC_SUBST(TERM_SETAF4, '')
+			AC_SUBST(TERM_SETAF6, '')
 	])
 ])
 
--- audacious-plugins-2.5.0/buildsys.mk.in~	2011-04-16 18:10:09.000000000 +0200
+++ audacious-plugins-2.5.0/buildsys.mk.in	2011-05-04 13:15:04.393333839 +0200
@@ -93,7 +93,6 @@
 
 MO_FILES = ${LOCALES:.po=.mo}
 
-.SILENT:
 .SUFFIXES:
 .SUFFIXES: .beam .c .c.dep .cc .cc.dep .cxx .cxx.dep .d .erl .lib.o .mo .m .m.dep .mm .mm.dep .o .plugin.o .po .py .pyc .xpm .S .S.dep
 .PHONY: all subdirs pre-depend depend install install-extra uninstall uninstall-extra clean distclean locales

